tests/integration/test_payments.py

```diff
@@ -155,3 +155,14 @@ def test_quote_payment_fee(self):
             assert quote_payment_fee.payment_type == "regular"
             assert quote_payment_fee.charge_type is None
 
+    def test_payments_delivery_date_error_response(self):
+        with Betamax(self.client.config.session) as betamax:
+            betamax.use_cassette('payments/delivery_date_error')
+            with pytest.raises(BadRequestError):
+                self.client.payments.payment_delivery_date(payment_date='2020-08-02', payment_type='priority', currency='USD', bank_country='CA')
```

Comment:
Why does the date ='2020-08-02' produces date error ?
Is it because it's in the past ?

Reply:
it was not a trading day (it is a Sunday)